raytracing/thorlabs.py

```diff
@@ -32,7 +32,15 @@ def __init__(self):
 
 class AC254_030_A(AchromatDoubletLens):
     def __init__(self):
-        super(AC254_030_A,self).__init__(fa=30.0,fb=22.9, R1=20.89, R2=-16.7, R3=-79.8,
+        """ As of May 31 2020, there is an error on the product web page at: 
```

Comment:
Inspired by my finding on Edmund Optics web site, I found this mistake on Thorlabs' web site.